guides/intro_to_git.md

```diff
@@ -0,0 +1,316 @@
+
+* To create a project from GitHub, select the "Your Workspace" option on the left hand side once in RStudio Cloud and select New Project on the far right of the top toolbar, in the Your Projects space.
+
+![](../images/new_project_git.png)
+
+* This will open a new dialogue that will ask for the Git Repository URL, this is the URL you copy, as described in [Step 2](01_intro_to_git.md#2-git-clone-copy-a-github-repository) of this guide. 
+* You will be prompted to enter your username (GitHub username) and your password, which is the access token created from the [Create Access token step](01_intro_to_git.md#creating-your-access-token-on-github).
+* Once the Git repository is deployed, you will notice the repository file structure on the right hand side panel.
+
+![](../images/project_structure.png)
+
+* Steps [3](01_intro_to_git.md#3-create-and-switch-to-a-new-branch), [4](01_intro_to_git.md#4-add-a-new-file) and [5](01_intro_to_git.md#5-commit-your-changes-follow-basic-git-command-workflow) will be the same, the only change being that instead of using your local terminal to input Git commands, you will be using the RStudio Cloud built-in terminal.
+
+![](../images/terminal.png)
+
+# General: How to submit a pull request
+
+**Pull request (PR)** is the application you submit on the GitHub repository, to announce to other collaborators working on the repository that you have a new change ready to be merged/part of the main build, the main branch of the repository. Then, either your collaborators or someone you assign will review the change and the branch you have been working on to determine whether the change is ready to be merged with the main branch. The equivalent term for GitLab users is Merge Request (MR).
+
+To submit a pull request:
+
+1. Simply head over to GitHub and to the respective repository
+
+2. Select Pull requests on the top bar menu. 
+
+3. Either select New Pull Request (big green button) on the right hand side or select Compare & pull request if you recently pushed new changes to the branch.
+
+4. Base ref will default to the main branch, head ref will be the branch you wish to merge with the main branch. You can change any of these options, there will be occasions you won't always aim to merge every branch to the main branch.
+
+5. Click Create Pull Request.
+
+6. Fill out any information related to the Pull request, title, description, any comments. On the rigth hand side you can assign yourself as the Assignee, and assign someone in your team to be the Reviewer. Then, select Create Pull Request.
+
+# General: How to accept a pull request
+
+Once you are assigned the pull request's reviewer, you should receive an email notification from GitHub with a link to the pull request's webpage. You can also view any active pull requests by clicking on Pull Requests on the top bar menu.
```

Comment:
Is it worth explaining the general process of reviewing, rather than listing the different views? I feel like if you are new to this, you don't necessarily need a description of what everything is, but you do want to know what to look at (e.g. Files changed)- I rarely look at commits or Checks

Reply:
ah I see what you mean now. Yeah, fair point, I think it might be worth splitting this section out from the guide into its own guide basically